scripts/mixpanel_traffic.py

```
@@ -0,0 +1,189 @@
+    (action_browse_reciters,    15),  # reciter lists
+    (action_browse_riwayahs,     5),  # rare
+    (action_404,                 2),  # rare error
+]
+ACTION_FNS, ACTION_WEIGHTS = zip(*ACTIONS)
+
+
+def burst(token: str, count: int) -> list[tuple[str, int]]:
+    """A burst of related calls — e.g. user browses reciters then fetches tracks."""
+    results = []
+    # Common pattern: list -> filter -> fetch tracks
+    if random.random() < 0.4:
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)